src/api/issues.js

```diff
@@ -18,12 +18,23 @@ const issues = (req, res) => {
         payload: {}
       });
     }
+
     const payload = JSON.parse(body)
-    .map((issue) => ({
-      id: issue.id,
-      name: issue.name,
-      number: issues.number
-    }));
+      .filter((issue) => issue.state === 'open')
```

Comment:
lol i added this line also and then saw that 'open' is default

https://developer.github.com/v3/issues/#list-issues-for-a-repository